src/h2olog/misc/gen_raw_tracer.py

```diff
@@ -683,16 +682,20 @@ def generate_cplusplus(context, output_file):
 
   handle_event_func = r"""
 void h2o_raw_tracer::do_handle_event(const void *data, int data_len) {
-  const h2olog_event_t *event = static_cast<const h2olog_event_t*>(data);
+  // The perf event data is not aligned, so we use a local copy to avoid UBSan errors.
+  // cf. https://github.com/iovisor/bpftrace/pull/1520
+  h2olog_event_t event;
+  assert(sizeof(event) <= data_len);
```

Comment:
nits: This line causes a compiler warning of  `warning: comparison of integer expressions of different signedness`.

```c++
assert(sizeof(event) <= static_cast<size_t>(data_len));
```

Reply:
Oops! Will fix it. Thanks.